index.html

```diff
@@ -569,6 +569,19 @@ <h4>Mandatory fields</h4>
             obvious from the TD.</p>
         </section>
 
+        <section>
+          <h4>Date format</h4>
+          <p class="rfc2119-assertion" 
+          id="profile-date-format-1">
+            All date and time values MUST use the format defined in [ISO 8601].
+          </p>
+          <p class="rfc2119-assertion" 
+          id="profile-date-format-2">
+            Whenever reasonable, all times SHOULD use UTC as the standard time zone.
```

Comment:
Arch call on 9.9.: We mandate UTC as a timezone using "Z" specifier.

Reply:
normative reference needs to added.

Reply:
Corner cases:
24 hour time corner case is ambiguous, i.e. 24:00, which is 00:00.
We will exclude 24:00
See corresponding section on the canonical form in the TD spec.